internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -276,10 +277,12 @@ func (ss *StageState) HumanString() string {
 	if transition == "" {
 		transition = empty
 	}
+	tree := treeprint.New()
+	stageString := fmt.Sprintf("%s\t%s\t%s", ss.StageName, status, transition)
+	tree = tree.AddBranch(stageString)
```

Comment:
Maybe instead of redefining the tree, we can rename it? 
```suggestion
	stageTree := tree.AddBranch(stageString)
```

Reply:
Doing it that way makes the root of the tree a dot, which I thought looked bad (ignore the wonky indentation, which I'll fix if you'd prefer the renaming+dot):
```
$ copilot pipeline status

Which application's pipeline status would you like to show? dinder-jr
Found pipeline: pipeline-dinder-jr-huanjani-dinder
Pipeline Status

Stage               Status              Transition
-----               ------              ----------
.
└── Source                       Succeeded           ENABLED
    └── SourceCodeFor-dinder-jr  Succeeded
.
└── Build           Succeeded           ENABLED
    └── Build       Succeeded
.
└── DeployTo-test                    Failed              ENABLED
    ├── CreateOrUpdate-backend-test  Succeeded
    ├── CreateOrUpdate-flask-test    Succeeded
    └── CreateOrUpdate-react-test    Failed
.
└── DeployTo-quiz     -                 ENABLED

Last Deployment

  Updated At    1 week ago
$
```

Reply:
oooh. i like your version better.